### PR TITLE
Tech debt: org and zone - remove hardcoded urls

### PR DESCRIPTION
**Pull request by NickKhalow**
## What does this PR change?

- Removes hardcoded urls
- Config to select environment located in `DynamicSceneLoaderSettings`
- Urls could be obtained from `IDecentralandUrlsSource`
- Keys of urls are located in enum `DecentralandUrl`
- DecentralandUrlsSource cached already concatenated urls,

Minor
- Fixes ExitApp logic

## How to test the changes?

1. Launch the explorer
2. Everything should work fine as it is, no feature introduced

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: Explorer/Assets/DCL/FeatureFlags/FeatureFlagOptions.cs [hunk 1]
```diff
@@ -19,20 +21,30 @@ public struct FeatureFlagOptions
...
-            Debug = false,
-            Hostname = "https://decentraland.zone",
-        };
+        private static FeatureFlagOptions NewZone() =>
+            new ()
+            {
+                AppName = "explorer",
+                URL = URLDomain.FromString("https://feature-flags.decentraland.zone"),
+                Debug = false,
+                Hostname = "https://decentraland.zone",
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/FeatureFlags/FeatureFlagOptions.cs:39]: `Hostname` should be resolved from `DecentralandUrl.Host` so we avoid a duplicate

File: Explorer/Assets/Scripts/ECS/StreamableLoading/NFTShapes/URNs/IURNSource.cs [hunk 2]
```diff
@@ -22,5 +21,8 @@ public Fake(URLAddress url)
             public URLAddress UrlOrEmpty(URN urn) =>
                 url;
         }
+
+        public static string BaseURL(DecentralandEnvironment environment) =>
+            $"https://opensea.decentraland.{environment.ToString()!.ToLower()}/api/v2/chain/{{chain}}/contract/{{address}}/nfts/{{id}}";
```
Comment (mikhail-dcl) [inline Explorer/Assets/Scripts/ECS/StreamableLoading/NFTShapes/URNs/IURNSource.cs:26]: It should be resolved from `DecentralandUrl.OpenSea`

**Comment by NickKhalow** [comment]
Changes done

**Review by mikhail-dcl — APPROVED** [review]

**Review by anicalbano — APPROVED** [review]
🟢 Build working as expected

**Pull request merged**